src/AppBridgeTheme.spec.ts

```diff
@@ -42,6 +61,11 @@ describe('AppBridgeThemeTest', () => {
         expect(appBridge.getProjectId()).toBe(PROJECT_ID);
     });
 
+    it('returns the hub id', () => {
+        const appBridge = new AppBridgeTheme(HUB_ID);
+        expect(appBridge.getProjectId()).toBe(HUB_ID);
```

Comment:
Shouldn't this be `getHubId()`?

```suggestion
        expect(appBridge.getHubId()).toBe(HUB_ID);
```